app/models.py

```diff
@@ -1332,7 +1331,7 @@ class Notification(db.Model):
         onupdate=datetime.datetime.utcnow)
     status = db.Column(
         'notification_status',
-        db.String,
+        db.Text,
```

Comment:
https://www.postgresql.org/docs/11/datatype-character.html
https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.Text

This allows for unlimited length strings. What currently goes in the status? I do not believe we should allow unlimited string length.

Reply:
Ditto what Kyle wrote.  You need to specify a max length.  i.e. "db.String(255)" or whatever number is appropriate.